JavaScript/YoutubeSearch/scripts/DOM.js

```diff
@@ -0,0 +1,167 @@
+//global variable declaration
+
+let pageIndex = -1;
+let items = new Array();
+let totalresult = 0;
+let elementsPerPage = 4;
+const url =
+  "https://www.googleapis.com/youtube/v3/search?key=AIzaSyCyjRf9SI3vh3FATvFJP1f2z6-0GEyaCCA&type=video&part=snippet&order=viewCount&maxResults=15&q=";
+let query = "";
+let nextPageToken = "";
+let previousPageToken = "";
+
+//initialize function
+(function init() {
+  headerDiv();
+  searchDiv();
+  displayDiv();
+  navigationDiv();
+})();
+
+//creating header
+function headerDiv() {
+  const header = document.createElement("header");
+  //creating h1 tag
+  const title = document.createElement("h1");
+  const titleText = document.createTextNode("Youtube Search");
+  title.appendChild(titleText);
+  header.appendChild(title);
+
+  const body = document.querySelector("body");
+  body.appendChild(header);
+}
+
+function searchDiv() {
+  //input tag
+  const searchBox = document.createElement("input");
+  searchBox.setAttribute("type", "text");
+  searchBox.setAttribute("id", "searchbox");
+  searchBox.setAttribute("placeholder", "search");
+
+  const searchButton = document.createElement("button");
+  searchButton.innerHTML = "&#128269;";
```

Comment:
The textContent property is great if you're only adding text, but if you're adding some third-party content as part of some additional markup, you'll still want to use innerHTML.

If you are using it for just adding HTML try to use textContent, as i am seeing in your code, i think you are adding some icon or something, what that number refers to, look into it if this works with textContent use it,